provider/testdata/data/json/props/exemplarchars/auxiliary@1/ru.json

```diff
@@ -2,15 +2,15 @@
   "CPInversionListStrList": {
     "cp_inv_list": [],
     "str_list": [
-      "а\\u0301",
-      "е\\u0301",
-      "и\\u0301",
-      "о\\u0301",
-      "у\\u0301",
-      "ы\\u0301",
-      "э\\u0301",
-      "ю\\u0301",
-      "я\\u0301"
+      "а́",
```

Comment:
This is more readable though

Reply:
We can't programatically determine whether some given diacritic is more readable or not. It's safer (and more explicit) to have them written out

Reply:
I'd say a diacritic actually combined with the character is always more readable than an escape sequence

Reply:
I'll punt on this for now since we should instead implement a proper parsing for UnicodeSet, for which this syntax of exemplar characters should be (should be?? is?? is not??) a subset of the UnicodeSet serialization.

Parsing correctly has been the hard part.  Creating a pretty print formatter to customize how we escape a certain set of characters (ex: non-BMP; gc != <...> ) in our JSON output is the easy part that we can do later once we agree on what that should look like.